task2/src/server.js

```diff
@@ -1,18 +1,29 @@
 import express from 'express'
 import router from './router.js'
 
-const server = express()
-server.use('/', router)
+const startServer = () => {
+      const server = express()
+      server.use('/', router)
```

Comment:
It is not in this PR, but in your code your are using "/user" as your endpoint. In rest apis resources (endpoints) are plurals. like `/users/someUserId`